src/sidebar/components/test/menu-keyboard-navigation-test.js

```diff
@@ -107,6 +113,13 @@ describe('MenuKeyboardNavigation', () => {
       wrapper.simulate('keydown', { key: 'Home' });
       assert.equal(document.activeElement.innerText, 'Item 1');
     });
+
+    it('does not throw an error when unmounting the component before the focus timeout finishes', () => {
+      const wrapper = createMenuItem({ visible: true });
+      wrapper.unmount();
+      clock.tick(1);
```

Comment:
I confirmed this test fails when commenting out the respective clearTimeout() call in the component